revolv/static/project.scss

```diff
@@ -77,3 +510,207 @@
     height: 300px;
     margin: 0 20px 20px 20px;
 }
+
+
+@media screen and (max-width: 1050px) {
+    .project-updates {
+
+        .small-12 {
+            padding-left: 0px;
+            padding-right: 0px;
+        }
+
+        .update-container {
+            margin: 10px 0px;
+
+            .date-container {
+                text-align: left;
+                margin-bottom: 50px;
+
+            }
+
+            .update-text {
+                font-weight: 500;
+                font-size: 1rem;
+            }
+        }
+    }
+}
+
+@media screen and (max-width: 800px) {
+
+    .dashboard {
+        .graphics-container {
+            .top-bar {
+                height: 40px;
+                top: 50%;
+
+                .container {
+
+                    .amount-donated {
+                        width: 40%;
+                        line-height: 40px;
+                        font-size: 0.7rem;
+                        margin: 0px 0px 0px 10px;
+
+                    }
+
+                    .days-left {
+                        line-height: 40px;
+                        font-size: 0.7rem;
+                        margin: 0px 10px 0px 0px;
+                    }
+                }
+            }
+
+            .bottom-bar {
+                height: 40px;
+                top: 60%;
+
+                .container {
+
+                    .amount-saved {
+                        line-height: 40px;
+                        font-size: 0.7rem;
+                        margin: 0px 0px 0px 10px;
+                    }
+
+                    .num-donors {
+                        line-height: 40px;
+                        font-size: 0.7rem;
+                        margin: 0px 10px 0px 0px;
+                    }
+                }
+            }
+
+            .svg-graphics-container {
+                width:0%;
```

Comment:
LOL tbh not sure why I did that - I'll reposition it in a way that makes more sense